Sources/OpenTelemetrySdk/Trace/SpanProcessor.swift

```diff
@@ -33,5 +33,12 @@ public protocol SpanProcessor {
 
     /// Processes all span events that have not yet been processed.
     /// This method is executed synchronously on the calling thread
-    func forceFlush()
+    func forceFlush(timeout: TimeInterval?)
```

Comment:
should the new argument be documented? Specifically, end users will appreciate some information on what does zero timeout means and how to make it infinite

Reply:
Of course it should, added